### ISSUE TITLE
Cannot create temporary file ... no such file or directory

### ISSUE THREAD

**Issue by GWRon** [bmx-ng/bmx-ng#93]
My dev computer in the office just got Windows 10 and I copied back my backup'd NG.

When I now try to compile (non-quick) it compiles some stuff - but then errors out:
```
[ 85%] Compiling:base.grid.bmx.release.win32.x64.c
Cannot create temporary file in C:\Users\username\AppData\Local\Temp\: No such file or directory
```
And yes, this "Temp" folder is not existing.
On my Windows 10 notebook this message does not appear - and there is a "Temp" folder in the user's AppData\Local.

The office computer is of course limited / with kind of restricted access, ... Maybe some   maintenance utility just removed the Temp-folder when it updated stuff / cleaned from unallowed files,...

Nonetheless I as the logged in user am allowed to manually create the "Temp" folder - done this and I was able to compile again.

TLDR: should bcc/bmk check if the "Temp"-folder exists and if it doesn't, try to recursively create it?

**Comment by GWRon** [comment]
Now tried to compile again - same error, folder gone again. So it seems the Maintenance Utility they use cleans the Temp from time to time (as long as you do no have a file open there).

**Comment by GWRon** [comment]
Reason of the issue was - as I assumed - the maintenance utility. It was migrated from Windows7 to Windows10 - and without further checks was cleaning the uers's local/temp folder by removing the folder instead of the files and folders in it.

Still I am not sure if bcc/bmk should try to create directories it wants to use - like the temp folder.
If not - just close the issue.

**Comment by GWRon** [comment]
@woollybah
If you want I could add directory checks to bcc/bmk.

If it is not wanted to happen at all, simply close the issue ;-)
